packages/xo-server/src/api/host.js

```diff
@@ -2,6 +2,46 @@ import { format } from 'json-rpc-peer'
 
 // ===================================================================
 
+export async function getSchedulerGranularity({ host }) {
+  let schedGran
+  try {
+    schedGran = await this.getXapi(host).getField('host', host.id, 'sched_gran')
```

Comment:
`getField` requires a ref instead of an ID. See https://github.com/vatesfr/xen-orchestra/blob/master/packages/xen-api/src/index.js#L302